Platform/MeteorlakeBoardPkg/AcpiTables/EcSsdt/Devices/Bat1Real.asl

```diff
@@ -151,6 +152,24 @@ Device (BAT1) {
       }
     }
   }
+
```

Comment:
If this part is not needed let's not include it

Reply:
done